docs/source/conf.py

```diff
@@ -67,26 +66,39 @@
 
 
 # The name of the Pygments (syntax highlighting) style to use.
-pygments_style = 'sphinx'
+pygments_style = 'friendly'
+pygments_dark_style = "monokai"
 highlight_language = 'python3'
 
 # -- Options for HTML output -------------------------------------------------
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'sphinx_rtd_theme'
-html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
+html_theme = 'furo'
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
 #
 html_theme_options = {
-    'collapse_navigation': False,
-    'display_version': False,
-    'logo_only': False,
-    'analytics_id': 'G-40DVRMX8T4',
```

Comment:
I hadn't realize, but we can't afford to drop this (this is what gives us good SEO on Google Site Manager)

Reply:
fixed by custom function (theme does not support)